kit/src/components/message/mod.rs

```diff
@@ -163,15 +162,18 @@ pub fn Message<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
         })
     });
 
-    // if markdown support is enabled, we will create it, otherwise we will just pass text.
-    let mut formatted_text = if cx.props.parse_markdown {
-        markdown(&text)
+    let loading_class = loading.then_some("loading").unwrap_or_default();
+    let remote_class = is_remote.then_some("remote").unwrap_or_default();
+    let order_class = if cx.props.order.is_some() {
```

Comment:
Wouldnt this be redundant due to `order` using `unwrap_or`? In this case, we could do `let order_class = order.to_string();`

Reply:
you're right